src/components/shell/shell.js

```diff
@@ -191,6 +192,10 @@ class Shell extends Component {
 
     if (this.state.nav && (e.keyCode === KeyMap.ENTER || e.keyCode === KeyMap.SPACE)) {
       this.unMuteFallback();
+      if (e.srcElement.contains(this._playerRef)) {
```

Comment:
why is the condition needed ?

Reply:
@SivanA-Kaltura  it seems that if you use the keyboard for other controls on the player it bubbles to the parent (the root of player), so e want to make sure we toggle play/pause when the event came from the video player element, as we don't want to do it if for example the user pressed the spacebar for seek forward button

Reply:
won't this catch every spacebar click on the root document as well though ?

Reply:
no, I checked and verified; the `event.srcElement` is different for the other controls..